.gitlab-ci.yml

```diff
@@ -104,3 +105,28 @@ release_integration:
     url: https://dss.integration.data.humancellatlas.org
   only:
     - master
+
+release_staging:
+  stage: release_staging
+  script:
+    - git clean -df
+    - git reset --hard
+    - for i in $(seq 1 10); do
+    -   if [[ "pending" != $(scripts/status.sh HumanCellAtlas dcp staging) ]]; then break; fi
+    -   echo "waiting for DCP Integration test to complete";
+    -   sleep 120;  # This loop will check status for 20 minutes and then quit
```

Comment:
Try testing more often (every 30s?). No harm in doing so and less delay if someone is waiting for the deployment to go through.